assignment-client/src/avatars/ScriptableAvatar.cpp

```diff
@@ -17,16 +17,18 @@
 
 #include <shared/QtHelpers.h>
 #include <AnimUtil.h>
+#include <AvatarHashMap.h>
 #include <ClientTraitsHandler.h>
 #include <GLMHelpers.h>
 #include <ResourceRequestObserver.h>
 #include <AvatarLogging.h>
 #include <EntityItem.h>
 #include <EntityItemProperties.h>
+#include <NetworkAccessManager.h>
```

Comment:
AvatarHashMap.h was dropped from ScriptEngine.h.  Unsure about NetworkAccessManager.h, but it may have been related to LimitedNodeList.h which was also dropped.